cremalab/sarah-todo
# Adasp 23 router

## Description

DESCRIPTION_HERE

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [ADASP-23](https://crema.atlassian.net/browse/ADASP-23)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] “/” shows the homepage a button that says “Get Started”
- [x]  “Get Started” button click takes user to “/todo”
- [x] “/todo” shows the tasks page where users can manage tasks





### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Open http://localhost:3000/
2. Click on the "Get Started" button. This should take you to the main tasks page.

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
